### ISSUE TITLE
DampedString instead of DampedSpring in Physics2DServer

### ISSUE THREAD

**Issue by pwab** [godotengine/godot#27921]
**Godot version:**

- Found in Godot 3.1 stable
- Confirmed for latest commit 5823b5d

**Issue description:**

I was reading the Physics2DServer implementation and found some strange naming conventions. It seems that there is a misspelling in the definition of the [Damped**Spring**Joint2D](https://docs.godotengine.org/en/3.1/classes/class_dampedspringjoint2d.html). The params are called `DAMPED_STRING_` instead of `DAMPED_SPRING_` (I assume that this is wrong because this param is used for the spring joint).

https://github.com/godotengine/godot/blob/3b697ce8d56ea7e73c202058f6c4d0e202142eae/servers/physics_2d_server.h#L565-L571

Not sure if this affects anything but I think straightforward naming conventions would be better for accurate code.

**Steps to reproduce:**
See here for respective code:

- https://github.com/godotengine/godot/search?q=DampedStringParam&unscoped_q=DampedStringParam

Docs:

- https://docs.godotengine.org/en/3.1/classes/class_physics2dserver.html?highlight=DampedStringParam#enum-physics2dserver-dampedstringparam
- http://docs.godotengine.org/en/3.1/classes/class_physics2dserver.html#class-physics2dserver-method-damped-spring-joint-create
- http://docs.godotengine.org/en/3.1/classes/class_physics2dserver.html#class-physics2dserver-method-damped-string-joint-get-param
- http://docs.godotengine.org/en/3.1/classes/class_physics2dserver.html#class-physics2dserver-method-damped-string-joint-set-param

**TLDR**

DampedS**t**ringParam and the corresponding functions should be renamed to DampedS**p**ringParam.
